includes/polling/applications/opengridscheduler.inc.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+use LibreNMS\RRD\RrdDefinition;
+
+$name = 'ogs';
+$app_id = $app['app_id'];
+$oid = '.1.3.6.1.4.1.8072.1.3.2.3.1.2.3.111.103.115';
+
+echo ' ' . $name;
+
+// get data through snmp
+$ogs_data = snmp_walk($device, $oid, '-Oqv', 'NET-SNMP-EXTEND-MIB');
```

Comment:
You can drop `, 'NET-SNMP-EXTEND-MIB'` as it's not needed when using numeric OID